cachey/cache.py

```diff
@@ -3,8 +3,8 @@
 from heapdict import heapdict
 import time
 
-def cost(nbytes, time):
-    return float(time) / nbytes / 1e9
+def cost(nbytes, time, zvalue=.5):
+    return float(time) / (nbytes if nbytes else zvalue) / 1e9
```

Comment:
@accurrently are you planning to change the use of the keyword arg here?

I recommend the following:

```suggestion
    return float(time) / (nbytes or 1) / 1e9
```